packages/shared/src/components/buttons/QuaternaryButton.tsx

```diff
@@ -9,11 +9,11 @@ import React, {
 import classNames from 'classnames';
 import { AllowedTags, Button, ButtonElementType, ButtonProps } from './Button';
 
-type QuandaryButtonProps = {
-  id: string;
-  reverse?: boolean;
-  responsiveLabelClass?: string;
-};
+export type QuaternaryButtonProps<TagName extends AllowedTags> =
```

Comment:
why do we  use this as a separate button again? 😅

Reply:
One word. Legacy 😓

Hopefully, we can spend some time in the future and understand whether we can actually remove this.